website/src/pages/use-cases/onboarding.tsx

```diff
@@ -0,0 +1,324 @@
+                    </div>
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-gradient-blue-mist">
+            <ContentSection className="my-7">
+                <div className="row flex-column flex-lg-row align-items-center justify-content-between">
+                    <div className="p-lg-0 col-lg-6 px-4">
+                        <h1 className="mb-4 font-weight-light">Developer onboarding is slow and expensive</h1>
+                        <p>
+                            Current tools and practices don't enable teams to onboard developers effectively or
+                            efficiently.
+                        </p>
+                        <p>
+                            <strong>What does that mean for you?</strong>
+                        </p>
+                        <ul>
+                            <li>New developers are left confused, struggling in front of an opaque codebase.</li>
+                            <li>
+                                Senior teammates need to break focus to answer questions while new devs wait to be
+                                unblocked.
+                            </li>
+                            <li>
+                                Existing developers onboarding to new projects struggle to maintain turnaround times as
+                                they learn unfamiliar parts of the codebase.
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="col-lg-5 mt-lg-0 mt-7">
+                        <h4 className="font-weight-bold text-center">Lunar makes every developer autonomous</h4>
+                        <Blockquote
+                            quote={`Sourcegraph makes it possible for us to enable every team to develop autonomous practices and solve cross-coding issues. 
+                            This autonomy is vital to ensure developers and their teams can accomplish their day-to-day work in isolation without being blocked.`}
+                            by="Bjørn Hal Sørensen, Web Architect at Lunar"
+                            logoHref="https://www.lunar.app/"
+                            logoImage="/external-logos/lunar.svg"
+                            border={true}
+                        />
+                    </div>
+                </div>
+            </ContentSection>
+        </div>
+
+        <ContentSection>
+            <div className="row mt-lg-0 mt-5 justify-content-center">
+                <div className="d-flex flex-column mt-lg-6 mt-4 w-100 mx-3">
+                    <h1 className="font-weight-light text-lg-center text-left mb-lg-6 mb-md-6">
+                        How Sourcegraph helps
+                    </h1>
+                </div>
+                <div className="pb-lg-5 pb-md-8 pb-5">
+                    <CustomCarousel items={items} autoAdvance={true} hideCarouselNav={true} smallPanel={true} />
```

Comment:
Seems like we need to make this stacked until the `lg` breakpoint. I'm still seeing the same issue from before.

![Screen Shot 2022-03-23 at 2 30 41 PM](https://user-images.githubusercontent.com/1733936/159772729-93e3125b-cd9d-432d-8e00-ea91bfdfb34c.png)

Reply:
Hey! Could you specify the issue a bit more? We don't have a design spec for the medium breakpoint, so all of the carousels look like this at `md`

Reply:
Yeah for sure! I would just make it look like the mobile version until we hit the `lg` breakpoint so that the individual items don't look crammed on the left.

Reply:
Updated! Had to add a different way of dealing with the different behaviors between these pages and code insights, but got something that works I think (on code insights, the carousel active item headers display on mobile, but they don't on these pages)

Reply:
Nice, looks good to me!